src/main/java/io/vertx/gsoc2023/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -96,6 +116,21 @@ public void start(Promise<Void> startFuture) throws Exception {
               .putHeader("content-type", "application/json")
               .end((new JsonObject().put("error", failure.getMessage()).toBuffer()));
           });
+      });*/
+
+      router.get("/realtime").handler(ctx -> {
+        ctx.request().toWebSocket()
+          .onSuccess(serverWS -> {
+            MessageConsumer<JsonObject> consumer = eb.consumer(serverWS.binaryHandlerID());
```

Comment:
If you use the binary handler id of the websocket, you don't need to register a consumer, there is one already registered automatically by Vert.x.
In this case, you should save the binary handler ID in a collection , and when a new quote is saved, send a message to address = binaryHandlerID

Reply:
Done Sir